x-pack/plugins/endpoint/server/routes/alerts/details/types.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
```

Comment:
super nit: there's an extra asterisk here

Reply:
@marshallmain Supposed to be there 😂  http://speakingjs.com/es5/ch29.html